src/Routers/routes/ConsultationRoutes.tsx

```
@@ -1,13 +1,14 @@
~import { Suspense, lazy } from "react";
~
~import Loading from "@/components/Common/Loading";
~import QuestionnaireResponseView from "@/components/Facility/ConsultationDetails/QuestionnaireResponseView";
~import EncounterQuestionnaire from "@/components/Patient/EncounterQuestionnaire";
~import TreatmentSummary from "@/components/Patient/TreatmentSummary";
~
 import { AppRoutes } from "@/Routers/AppRouter";
 import { EncounterShow } from "@/pages/Encounters/EncounterShow";
 import { PrintPrescription } from "@/pages/Encounters/PrintPrescription";
+import { PrintQuestionnaireQuestionnaireResponses } from "@/components/Facility/ConsultationDetails/PrintQuestionnaireQuestionnaireResponses";
```

ESLint

[error] Delete `";⏎import·{·PrintQuestionnaireQuestionnaireResponses·}·from·"@/components/Facility/ConsultationDetails/PrintQuestionnaireQuestionnaireResponses`

(prettier/prettier)